test/examples/example_quivers.m

```diff
@@ -0,0 +1,70 @@
+%% Quiver calculations
+% These are calculations for the quiver dimensions as implemented in MATLAB
+% (HG1) as in the |quiver.m| function.
+%
+% For HG2 and Octave, the situation might be different.
+%
+% A single quiver is defined as:
+%
+%                   C
+%                    \
+%                     \
+%  A ----------------- B 
+%                     /
+%                    /
+%                   D
+%
+% To know the dimensions of the arrow head, MATLAB defines the quantities
+%  alpha = beta = 0.33 that determine the coordinates of C and D as given below.
+
+clc; 
+clear variables;
+close all;
+
+%% Parameters
+syms x y z u v w alpha beta epsilon real
```

Comment:
It's not practical to get rid of this dependency (i.e. I used this script to determine the angle of the arrow heads, which is something one can do for specific values of `x,y,z,u,v,w` without that toolbox).
I will write some fall-back code such that the script also runs without the toolbox at the cost of producing less generic values.